exasol_sphinx_github_pages_generator/parser.py

```diff
@@ -16,4 +16,6 @@ def __init__(self):
         parser.add_argument('--source_branch', type=str,
                             default="main",
                             help='current branch')
-        self.args = parser.parse_args()
+        parser.add_argument('--source_dir', type=str, default="./doc/", help="The doc directory your documentation fildes adn conf.py reside in")
+        parser.add_argument('--module_path', type=str, default="../module_name", help="The path to the module the docu is beeing generated for")
```

Comment:
Does the default value make sense?

Reply:
At the moment no default value makes sense here. I set these to clarify the usage a little, but i can also take them out.

Reply:
Ok, maybe add a TODO to not forget to change it later

Reply:
done